scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/task/internal/InternalTask.java

```diff
@@ -85,6 +90,18 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 public abstract class InternalTask extends TaskState {
 
+    @XmlTransient
+    public static transient final TaskLogger logger = TaskLogger.getInstance();
+
+    @XmlTransient
+    private static transient HashSet<String> authorizedSelectionScripts = null;
+
+    @XmlTransient
+    private static final transient long DEFAULT_AUTHORIZED_SCRIPT_LOAD_PERIOD = 60 * 1000;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Cast one of the operands of this multiplication operation to a "long". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS2184)